code/game/machinery/computer/extraction_cargo.dm

```diff
@@ -63,6 +63,13 @@
 		new /datum/data/extraction_cargo("S-Corp Brand Soda SMG ",		/obj/item/gun/ego_gun/sodasmg,										400, CAT_SCORP) = 1,
 		new /datum/data/extraction_cargo("S-Corp Brand Shrimp Squad ",	/obj/item/grenade/spawnergrenade/shrimp,							800, CAT_SCORP) = 1,
 
+		//Resources - Raw PE, ETC. Abnochem stuff goes here too. This is one use items to further LC13 systems
+		new /datum/data/extraction_cargo("Blue Filter ",				/obj/item/refiner_filter/blue,										10, CAT_RESOURCE) = 1,
+		new /datum/data/extraction_cargo("Green Filter ",				/obj/item/refiner_filter/green,										10, CAT_RESOURCE) = 1,
+		new /datum/data/extraction_cargo("Red Filter ",					/obj/item/refiner_filter/red,										10, CAT_RESOURCE) = 1,
+		new /datum/data/extraction_cargo("Yellow Filter ",				/obj/item/refiner_filter/yellow,									10, CAT_RESOURCE) = 1,
+		new /datum/data/extraction_cargo("Raw PE Box ",					/obj/item/rawpe,													50, CAT_RESOURCE) = 1,
```

Comment:
I don't think these should be purchasable, considering _we make these in house_. I feel it'd be much more interesting if they spawned at a high chance on good results and a "normal" chance on neutral results, or something similar to that.

Reply:
Clarify im referring to specifically just the PE boxes here.

Reply:
This is mostly a bandaid for now, if nothing else I'll add in a little machine outside extraction that does virtually the same thing.